interactions/models/discord/user.pyi

```diff
@@ -131,6 +133,8 @@ class Member(FakeUserMixin):
     @property
     def display_avatar(self) -> Asset: ...
     @property
+    def avatar_url(self) -> str: ...
+    @property
```

Comment:
There's no need to add this since `Member` inherits from the fake user mixin.

Reply:
Oh yes, that's just me not paying attention, thank you!